src/coreComponents/mainInterface/ProblemManager.cpp

```diff
@@ -186,6 +186,13 @@ void ProblemManager::parseCommandLineInput()
   OutputBase::setOutputDirectory( outputDirectory );
 
   string & inputFileName = commandLine.getReference< string >( viewKeys.inputFileName );
+
+  for( string xmlFile : opts.inputFileNames )
+  {
+    Path::pathPrefix() = getAbsolutePath( xmlFile );
```

Comment:
Having `Path::pathPrefix` returning a mutable reference to the global variable looks smells wrong.
I know it's not your PR, but could you add a `Path::setPathPrefix` and remove the mutable calls here and there (as well as the mutable version of the function)?

Reply:
Oops, I did not see that. He should not have modified `Path::pathPrefix()` here as it was not his intention (he just would like to simplify the path (get rid of `./`, `../`).
A local variable would be sufficient here, but getting rid of the mutable calls would be nice too!

@TotoGaz Under the hood, when multiple input XML are used, it create a `composite_input.xml` file, but the working dir is still the current directory from where GEOS was started.